src/main/java/app/gpx_animator/ui/TrackSettingsPanel.java

```diff
@@ -355,6 +413,7 @@ public void setConfiguration(final TrackConfiguration c) {
         trimGpxStartSpinner.setValue(c.getTrimGpxStart());
         trimGpxEndSpinner.setValue(c.getTrimGpxEnd());
         trackIconComboBox.setSelectedItem(c.getTrackIcon());
+        flipIcon.setSelected(c.getFlipIcon());
```

Comment:
I think the value of `inputIconFileSelector` should be set here, like a few lines above with the `inputGpxFileSelector` as an example. Otherwise loading a saved configuration will lose the custom icon.